Update set-up-bigquery-oauth.md (#7298)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/manage-access/set-up-bigquery-oauth.md

Now that you have an OAuth app set up in BigQuery, you'll need to add the client ID and secret to <Constant name="cloud" />. To do so:

>  - Navigate back to the connection details page, as described in [Locate the redirect URI value](#locate-the-redirect-uri-value)

Reviewer: ```suggestion
 1. Navigate back to the Connection details page, as described in [Locate the redirect URI value](#locate-the-redirect-uri-value).
```

Reviewer: Converted lines 65 - 67 to numbered steps as they should be done in order.

>  - Add the client ID and secret from the BigQuery OAuth app under the **OAuth 2.0 Settings** section

Reviewer: ```suggestion
 2. Add the client ID and secret from the BigQuery OAuth app under the **OAuth 2.0 Settings** section.
```

>  - Enter in the BigQuery token uri which is by default `https://oauth2.googleapis.com/token`

Reviewer: ```suggestion
 3. Enter the BigQuery token URI. The default value is `https://oauth2.googleapis.com/token`. 
```
